script/test

```diff
@@ -30,4 +30,6 @@ export ARM_CLIENT_SECRET=
 export ARM_TENANT_ID=
 export ARM_SUBSCRIPTION_ID=
 
-nosetests --with-no-network "$@"
+export PYTHONPATH=.:$PYTHONPATH
+
+pytest --disable-network tests.py
```

Comment:
@ross I needed to add tests.py explicitly here to make `script/cibuild` and `script/test work` -- I cannot tell, why this is not the case for https://github.com/octodns/octodns/pull/851 where there still is `"$@"` in place.
Maybe you could elaborate on that?
(Just like `script/cibuild` I just call `script/test` without parameters)

Reply:
`pytest` doesn't automatically find things in `test.py`. It wants them to live in files named `test_XXX.py`. I just moved and renamed the file in https://github.com/octodns/octodns-azure/pull/3/files#diff-765646ab9779321a281e380c28de3d858acc6a4a92da8ce9dfbe23fbedd86034 and have made a similar change to octodns-template. We should probably do that here so that it lines up with the others (or at least what they'll be in the near future.)